build/commands/scripts/commands.js

```diff
@@ -188,12 +188,16 @@ program
 
 program
   .command('pull_l10n')
+  .option('--service <service>', 'Service to use: Transifex or Crowdin')
+  .option('--channel <channel>', 'Release|Beta|Nightly for Crowdin, Release for Transifex')
   .option('--grd_path <grd_path>', `Relative path to match end of full GRD path, e.g: 'generated_resources.grd'.`)
   .option('--debug', `Dumps downloaded content for one language into TransifexCurrent.txt file in the temp directory.`)
```

Comment:
Does this description (and underlying functionality) need to be updated to use the appropriate service name?

Reply:
Fixed in [13140e2](https://github.com/brave/brave-core/pull/27005/commits/13140e28ea7cd53e4056067f06938b892d409c16)